src/main/java/uk/gov/hmcts/reform/sscs/extractor/DwpResponseReceivedDateExtractor.java

```diff
@@ -23,8 +33,8 @@
                 return Optional.of(event.getValue().getDateTime());
             }
         }
-
-        return Optional.ofNullable(caseData.getDwpResponseDate()).map(date -> ZonedDateTime.parse(date + "T00:00:00Z").toInstant().atZone(ZoneId.of(ZONE_ID)));
+        LocalTime time = LocalTime.MIN.plusSeconds(rand.nextInt(60 * 60 * 1000));
+        return Optional.ofNullable(caseData.getDwpResponseDate()).map(date -> ZonedDateTime.parse(date + "T" + time.toString() + "Z").toInstant().atZone(ZoneId.of(ZONE_ID)));
```

Comment:
question: Intention here is to give a randomness to the time element DWP Response Date which will vary within what bounds?

Reply:
Yes, at the moment I bounds the times 24 hours